contrib/src/main/java/com/datatorrent/contrib/parser/CsvParser.java

```diff
@@ -62,247 +71,193 @@
  * @since 3.2.0
  */
 @InterfaceStability.Evolving
-public class CsvParser extends Parser<String, String>
+public class CsvParser extends Parser<byte[], KeyValPair<String, String>>
```

Comment:
Should this be renamed to DelimitedParser rather than CsvParser.
I believe it can work on delimited strings, correct?